src/components/ItemFilters/ItemFilters.tsx

```diff
@@ -99,57 +85,42 @@ const FiltersContainer = ({
     submitFilters([year], "year")
   }
 
-  return (
-    <>
-      <Box
-        className={styles.filtersContainer}
-        sx={{
-          display: "flex",
-          flexDirection: isLargerThanMedium ? "row" : "column",
-        }}
-        mb="m"
-      >
-        <Box className={filterGroupClassName} ref={ref}>
-          <Label
-            id="filters-label"
-            htmlFor="item-filters"
-            fontWeight="bold"
-            data-testid="filters-label"
-          >
-            Filter By
-          </Label>
-          <Box
-            sx={{
-              display: "flex",
-              flexDirection: isLargerThanLarge ? "row" : "column",
-            }}
-            gap="nypl-s"
-            id="item-filters"
-          >
-            {filterData.map((field: ItemFilterData) => (
-              <ItemFilter
-                isOpen={whichFilterIsOpen === field.field}
-                setWhichFilterIsOpen={setWhichFilterIsOpen}
-                key={field.field}
-                itemFilterData={field}
-                appliedFilters={appliedFilters}
-                submitFilters={submitFilters}
-              />
-            ))}
-          </Box>
-        </Box>
-        <Box className={filterGroupClassName} minWidth={440}>
-          <Label
-            id="year-filter-label"
-            htmlFor="year-filter"
-            fontWeight="bold"
-            data-testid="year-filter-label"
-          >
+  // TODO: Replace this with actual filter data
+  const filterCheckboxGroups: FilterCheckboxGroup[] = [
+    {
+      id: "location",
+      name: "Location",
+      items: [{ id: "item-id", name: "Test" }],
+    },
+    {
+      id: "format",
+      name: "Format",
+      items: [],
+    },
+    {
+      id: "status",
+      name: "Status",
+      items: [],
+    },
+  ]
+
+  // function for renderChildren prop of FilterBarInline
+  const filterBarContent = () => {
+    return (
+      <>
+        <MultiSelectGroup
+          id="item-filters"
+          labelText="Filter by"
+          renderMultiSelect={() => filterCheckboxGroups.map(renderMultiSelect)}
```

Comment:
The render prop returns two props, `isBlockElement` and `multiSelectWidth`. You should be able to pass `multiSelectWidth` to each `MultiSelect`'s `width` prop. It's hardcoded to "fitContent" below.